presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -498,6 +500,8 @@ void PrestoServer::initializeVeloxMemory() {
     }
     cache_ = cache::AsyncDataCache::create(allocator_.get(), std::move(ssd));
     cache::AsyncDataCache::setInstance(cache_.get());
+    PRESTO_STARTUP_LOG(INFO)
```

Comment:
Message should be printed before initialization of cache?

Reply:
Yeah, better. thanks!